src/components/service/serviceUtils.ts

```diff
@@ -17,13 +17,21 @@ export const parseStorageKey = (input: string): StorageKey => {
     if (input.length === 64) {
       return bytes.Bytes.parseBytes(input, 31);
     }
+    if (input.length === 50) {
```

Comment:
Apologies for the mistake, these inputs should be 48 characters long and we should expand them with 18 zeros.